src/components/ShoppingBag/ListItemQuantityControls.tsx

```diff
@@ -0,0 +1,37 @@
+import Image from "next/image";
+import { Plus, Minus, Trash2 } from "lucide-react";
+
+import { Input } from "@/components/ui/input";
+import { Button } from "@/components/ui/button";
+
+import { type ShoppingListItem as ListItem } from "./types";
+
+/*
+Read about the accessibility approach here
+https://lucide.dev/guide/advanced/accessibility#on-icon-buttons
+*/
+
+export const ListItemQuantityControls = ({ quantity }: ListItem) => {
+  const buttonClass = "[&_svg]:size-5";
+  return (
+    <div className="flex gap-2">
+      <Button variant="ghost" size="icon" className={buttonClass}>
+        <Minus />
+        <span className="sr-only">Remove one of this item</span>
```

Comment:
nice that you added support for screen readers, i forgot that tailwind has a ready class for it